src/doc/python_scripting/functions.rst

```diff
@@ -10110,8 +10113,8 @@ SetPrecisionType
   SetPrecisionType(typeAsString)
 
 
-typeAsInt : double
-    Precision type specified as an integer. 0 = float 1 = native 2 = double
+typeAsInt : integer
```

Comment:
This scares me, since someone thought it should be a double. But everything I can find suggests that it is in fact an integer.

Reply:
I looked at the code and tested it as well. It takes the following args:

0, "Float"
1, "Native"
2, "Double"

The capitalization matters. The default is "Native".